themes/default/content/what-is/what-is-platform-engineering.md

```diff
@@ -77,8 +68,16 @@ Elkjøp Nordic is the leading consumer electronics retailer in the Nordics. The
 
 Washington Trust Bank modernized its software development and infrastructure practices since migrating to Azure and adopting infrastructure as code. It enables developers with self-service infrastructure components, prevents developers from deploying forbidden resources with CrossGuard policies, and uses automation to save time and effort.
 
-## Choosing an infrastructure as code framework
+## Conclusion
+
+Given platform engineering's strong focus on infrastructure management, automation, and standardization, platform engineering teams need a solution that enables them to adhere to the principles of platform engineering while empowering their developers and development teams to leverage the cloud in a secure, scalable, reliable, and consistent way.
+
+Pulumi's solution for platform teams encompasses all of the key needs described in this article:
 
-Choosing an infrastructure as code framework is an important foundation of a platform engineering strategy because it impacts how infrastructure is modeled, tested, distributed, and deployed across an organization. In particular, the choice of IaC language affects infrastructure’s usability, reusability, and capacity for scale. General-purpose languages provide major advantages because they are widely adopted already, have more flexibility and expressiveness than DSLs, and come with rich ecosystems of tools and frameworks that increase productivity. Last but not least, it’s much easier for developers to use IaC when it’s written in languages they already know.
+1. A strong infrastructure as code tool that supports automation as the default approach, with support for programming languages that teams are already familiar with  (including TypeScript, Python, Go, C#, Java, and YAML).
```

Comment:
the part i think is missing is page 3 here:  https://docs.google.com/document/d/19i93DNN7XnRzjIXz-4juJ2Kwyq70BI6_m3XjaeeW_DA/edit

i want to present our point of view of all the requirements and how the differnet layers are constructed

Reply:
what doesn't make sense is we don't use this diagram anywhere - https://www.pulumi.com/images/product/platform-teams-diagram.png

yes, we are talking generically about platform engineering. we should define it in the general, but then we should present our POV.

Reply:
I've added the diagram and rewritten the conclusion to more closely mirror the messaging document (and to tie back to the requirements described earlier in the page).